README.md

```diff
@@ -64,93 +64,80 @@ For other Linux distributions, you might need to handle the dependency issue by
 
 ### help
 
-Show usage info
+Show usage info:
 
 ```
-[root@c910f03c05k21 xcat-inventory]# xcat-inventory help
-usage: xcat-inventory [--debug] [-v] [-V] <subcommand> ...
-
-xCAT inventory management tool
-
-Positional arguments:
-  <subcommand>
-    export       Export the inventory data from xcat database
-    help         Display help about this program or one of its subcommands.
-    import       Import inventory file to xcat database
-
-Optional arguments:
-  --debug        Prints debugging output into the log file (not implemented
-                 yet).
-  -v, --verbose  Prints verbose output (not implemented yet).
-  -V, --version  Shows the program version and exits.
-
-See "xcat-inventory help COMMAND" for help on a specific command.
+[root@c910f03c05k21 inventory]# xcat-inventory help
```

Comment:
I think the command prompts should not be part of the output.  

should just be like this
```
# xcat-inventory help
```

Reply:
removed in https://github.com/xcat2/xcat-inventory/pull/37